Bindings/Python/__init__.py

```diff
@@ -1,15 +1,18 @@
 import sys
 import os
 if (sys.platform.startswith('win')):
-      curFolder = os.path.dirname(os.path.realpath(__file__))
-      os.add_dll_directory(curFolder)
-      # in install env, add relative path from init.py to bin, in build environment, we also add "../../../../Release"
-      install_path = os.path.join(curFolder, "../../../bin")
-      if (os.path.exists(install_path)):
-          os.add_dll_directory(install_path)
-      dev_path = os.path.join(curFolder, "../../../../Release")
-      if (os.path.exists(dev_path)):
-          os.add_dll_directory(dev_path)
+    curFolder = os.path.dirname(os.path.realpath(__file__))
+    os.add_dll_directory(curFolder)
+    # when installed locally via "python -m pip install ."
+    if os.path.isfile(os.path.join(curFolder, 'opensim-cmd.exe')):
```

Comment:
Add a comment to note that `os.environ['PATH']` only sets the path for the Python environment.